src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.vbproj

```diff
@@ -666,16 +669,11 @@
     <Folder Include="Resources\Common\" />
   </ItemGroup>
   <Import Project="..\..\build\Targets\VSL.Imports.targets" />
-
-
   <!-- We don't generate yet generate pkgdef, but we want it to end up in VSIX,
        implement the right output group so that ProjectSystemSetup pulls it in -->
-  <Target Name="PkgdefProjectOutputGroup"
-        Outputs="@(PkgdefOutputGroupOutput)">
-
+  <Target Name="PkgdefProjectOutputGroup" Outputs="@(PkgdefOutputGroupOutput)">
```

Comment:
Looks like this was auto-formatted when I added a new file, we should stop doing that: https://github.com/dotnet/roslyn/issues/11245.